submitit/core/core.py

```diff
@@ -485,6 +487,12 @@ def stderr(self) -> tp.Optional[str]:
             return "\n".join(stderr_not_none)
         return self._get_logs_string("stderr")
 
+    def async_job(self) -> "AsyncJobProxy[R]":
```

Comment:
other options:
- asynchronous (a like the word event though I cant spell it, and it's not more succinct...)
- asyncio (might be clumsy since it's the name of a standard library, but since one needs to use it along, maybe that's fine)

`async_job` to me refers to a full job, which you seem to mock with `getattr` and stuff so it's fine as well, but if we remove it maybe `async_proxy` would be more accurate.

Reply:
called it `awaitable()` but it looks like I forgot to push until today :-/